test/livebook/runtime/dependencies_test.exs

```diff
@@ -206,10 +206,13 @@ defmodule Livebook.Runtime.DependenciesTest do
              ) ==
                {:error,
                 """
-                ** (SyntaxError) nofile:2:2: syntax error before: ','
-                    |
-                  2 | [,1]
-                    |  ^\
+                ** (SyntaxError) invalid syntax found on nofile:2:2:
+                    \e[31merror:\e[0m syntax error before: ','
+                    │
+                  2 │ [,1]
+                    │ \e[31m ^\e[0m
+                    │
+                    └─ nofile:2:2\
```

Comment:
double-checking, is this how we want to handle these tests? ANSI escape codes add some noise maybe we clear them out somehow?

Reply:
You can extract it to TestHelpers from this:

https://github.com/livebook-dev/livebook/blob/4f38fea5cd36f662a9f89470593d25b4624c2c3a/test/livebook/runtime/evaluator_test.exs#L1378-L1390